v21.1/deploy-cockroachdb-with-kubernetes-openshift.md

```diff
@@ -0,0 +1,330 @@
+	cockroach-operator-65c4f6df45-h5r5n   1/1     Running   0          6m11s
+	crdb-client-secure                    1/1     Running   0          14m
+	crdb-tls-example-0                    1/1     Running   0          3m15s
+	crdb-tls-example-1                    1/1     Running   0          103s
+	crdb-tls-example-2                    1/1     Running   0          89s
+	~~~
+
+## Step 5. Use the CockroachDB SQL client
+
+1. Start the CockroachDB [built-in SQL client](cockroach-sql.html) from the client pod:
+
+	{% include copy-clipboard.html %}
+	~~~ shell
+	oc exec -it crdb-client-secure -- ./cockroach sql --certs-dir=/cockroach/cockroach-certs/ --host=crdb-tls-example-public
+	~~~
+
+	~~~
+	#
+	# Welcome to the CockroachDB SQL shell.
+	# All statements must be terminated by a semicolon.
+	# To exit, type: \q.
+	#
+	# Server version: CockroachDB CCL v20.2.8 	(x86_64-unknown-linux-gnu, built 2021/04/23 13:54:57, go1.13.14) (same version as client)
+	# Cluster ID: 0813c343-c86b-4be8-9ad0-477cdb5db749
+	#
+	# Enter \? for a brief introduction.
+	#
+	root@crdb-tls-example-public:26257/defaultdb>
+	~~~
+
+	Now you can run SQL commands against the cluster.
+
+	{% include {{ page.version.version }}/orchestration/kubernetes-basic-sql.md %}
+
+**Note:** If you can't access the SQL client, this may be related to your `--certs-dir` or `--host` flags. 
+
+1. Shell into the client pod and check for the necessary certs in the `--certs-dir` directory:
+
+	{{site.data.alerts.callout_success}}
+	You can also access the client pod by selecting it on the **Pods** page and clicking **Terminal**.
+	{{site.data.alerts.end}}
+
+	{% include copy-clipboard.html %}
+	~~~ shell
+	oc exec -it crdb-client-secure sh
+	~~~
+
+	{% include copy-clipboard.html %}
+	~~~ shell
+	# cd /cockroach/cockroach-certs
+	# ls
+	~~~
+
+	~~~
+	ca.crt           client.root.key
+	client.root.crt
+	~~~
+
+1. Check the name of the `public` service to use with the `--host` flag:
+
+	{% include copy-clipboard.html %}
+	~~~ shell
+	oc get services
+	~~~
+
+	{% include copy-clipboard.html %}
+	~~~ shell
+	NAME                      TYPE        CLUSTER-IP       EXTERNAL-IP   PORT(S)              AGE
+	crdb-tls-example          ClusterIP   None             <none>        26257/TCP,8080/TCP   14m
+	crdb-tls-example-public   ClusterIP   172.25.180.197   <none>        26257/TCP,8080/TCP   14m
+	~~~
+
+## Step 6. Access the DB Console
+
+To access the CockroachDB cluster's [DB Console](ui-overview.html):
+
+1. On secure clusters, [certain pages of the DB Console](ui-overview.html#db-console-access) can only be accessed by `admin` users.
+
+	Start the CockroachDB [built-in SQL client](cockroach-sql.html) from the client pod:
+
+	{% include copy-clipboard.html %}
+	~~~ shell
+	oc exec -it crdb-client-secure -- ./cockroach sql --certs-dir=/cockroach/cockroach-certs/ --host=crdb-tls-example-public
+	~~~
+
+1.  Assign `roach` to the `admin` role (you only need to do this once):
```

Comment:
Should there be a step before this where we create the user `roach` and set a password? Then this line would make sense in making reference to the username, and the later line would also make sense:

> log in with the username and password you created earlier.

Looks like we have the same issue here https://www.cockroachlabs.com/docs/v20.2/orchestrate-a-local-cluster-with-kubernetes.html#step-4-access-the-db-console probably based on the way the example was ported over from here https://www.cockroachlabs.com/docs/v20.2/secure-a-cluster.html#step-3-use-the-built-in-sql-client where we *do* have a step to `CREATE USER max WITH PASSWORD 'roach';`

Although in this case, we're using `roach` as the user, not the password.

An aside, I've never liked using `max` as a user name because it also has the meaning 'maximum' and could be confusing to see in a code block.